### PR TITLE
Minor catkinization fixes for python scripts.

### PR DESCRIPTION
**Pull request by ahendrix**
Confirmed building and installing properly on Hydro, with these fixes.

### CODE REVIEW

**Comment by davetcoleman** [comment]
Thanks for doing this Austin!

File: controller_manager/CMakeLists.txt [hunk 1]
```diff
@@ -26,6 +26,8 @@ else()
     include/controller_manager/controller_loader_interface.h
     include/controller_manager/controller_loader.h)
 
+  add_dependencies(${PROJECT_NAME} ${catkin_EXPORTED_TARGETS})
```
Comment (jbohren) [inline controller_manager/CMakeLists.txt:29]: @ahendrix This line is breaking my build on groovy and hydro...

Comment (jbohren) [inline controller_manager/CMakeLists.txt:29]: Fix here: https://github.com/willowgarage/ros_control/pull/65

Comment (ahendrix) [inline controller_manager/CMakeLists.txt:29]: That sounds fishy; this is according to the spec. How are you running your build, and what error message are you seeing?

**Pull request merged**